ovs/proto_trace_test.go

```diff
@@ -27,17 +27,17 @@ func Test_UnmarshalText(t *testing.T) {
 	}{
 		{
 			name: "action output port",
-			output: `Flow: tcp,in_port=3,vlan_tci=0x0000,dl_src=00:00:00:00:00:00,dl_dst=00:00:00:00:00:00,nw_src=192.0.2.2,nw_dst=0.0.0.0,nw_tos=0,nw_ecn=0,nw_ttl=0,tp_src=0,tp_dst=22,tcp_flags=0
+			output: `Flow: tcp,in_port=LOCAL,vlan_tci=0x0000,dl_src=00:00:00:00:00:00,dl_dst=00:00:00:00:00:00,nw_src=192.0.2.2,nw_dst=0.0.0.0,nw_tos=0,nw_ecn=0,nw_ttl=0,tp_src=0,tp_dst=22,tcp_flags=0
```

Comment:
I don't fully grok why these tests were changes instead of having additional ones. Care to elaborate?

Reply:
No good reason. I can add a new test.